app/javascript/mastodon/actions/compose.js

```diff
@@ -11,6 +12,9 @@ import {
   refreshPublicTimeline,
 } from './timelines';
 
+let CancelToken = axios.CancelToken;
+let _CancelFetchComposeSuggestionsAccounts;
```

Comment:
What is this underscore?

Reply:
Just a sign that this is to remain private to this module. But this convention doesn't appear to be used widely/consistently within Mastodon, so I guess I can drop it (and also start with a lowercase).

Reply:
I'm for dropping this because the code knows it, even without such a symbol. The code will not allow to use the variable out of this module.
If i recall correctly, underscore as prefix is seen in some components of Mastodon, but that could make sense because the code doesn't know a variable is private, and allows to use it out of the component (if `ref` is there.)